uui/components/tables/DataTableHeaderCell.tsx

```diff
@@ -16,78 +22,107 @@ import { ReactComponent as OpenedDropdownIcon } from '@epam/assets/icons/navigat
 
 interface DataTableHeaderCellState {
     isDropdownOpen: boolean;
 }
 
-export class DataTableHeaderCell<TItem, TId> extends React.Component<DataTableHeaderCellProps<TItem, TId> & DataTableHeaderCellMods, DataTableHeaderCellState> {
+export interface DataTableHeaderCellModsOverride {
+}
+
+export class DataTableHeaderCell<TItem, TId> extends
+    React.Component<
+    DataTableHeaderCellProps<TItem, TId> & Overwrite<DataTableHeaderCellMods,
+    DataTableHeaderCellModsOverride>,
+    DataTableHeaderCellState
+    > {
     state: DataTableHeaderCellState = {
         isDropdownOpen: null,
     };
 
-    getTextStyle = () => {
-        if (this.props.textCase === 'upper') return css.upperCase;
-        return css['font-size-14'];
-    };
-
     getTooltipContent = (column: DataColumnProps<TItem, TId>) => (
-        <div className={ css.cellTooltipWrapper }>
-            <Text fontSize="14" fontWeight="600" cx={ css.cellTooltipText }>{ column.caption }</Text>
-            { column.info && <Text fontSize="12" cx={ css.cellTooltipText }>{ column.info }</Text> }
+        <div className={ cx(css.cellTooltipWrapper, uuiDataTableHeaderCell.uuiTableHeaderCaptionTooltip) }>
+            <Text cx={ [css.cellTooltipText, css.tooltipCaption] }>
+                { column.caption }
+            </Text>
+            { column.info && (
+                <Text cx={ [css.cellTooltipText, css.tooltipInfo] }>
+                    { column.info }
+                </Text>
+            ) }
         </div>
     );
 
     getColumnCaption = () => {
         const renderTooltip = this.props.column.renderTooltip || this.getTooltipContent;
+        const captionCx = [
+            css.caption,
+            this.props.textCase === 'upper' && css.upperCase,
+            uuiDataTableHeaderCell.uuiTableHeaderCaption,
+            settings.sizes.dataTable.header.row.cell.truncate.includes(this.props.size) && css.truncate,
+        ];
 
         return (
-            <div className={ cx(css.captionWrapper, css['align-' + this.props.column.textAlign], uuiDataTableHeaderCell.uuiTableHeaderCaptionWrapper) }>
+            <div
+                className={ cx(css.captionWrapper, css['align-' + this.props.column.textAlign], uuiDataTableHeaderCell.uuiTableHeaderCaptionWrapper) }
+            >
                 <Tooltip
                     placement="top"
                     color="inverted"
                     content={ renderTooltip(this.props.column) }
                     cx={ css.cellTooltip }
                     openDelay={ 600 }
                 >
-                    <Text key="text" lineHeight="30" fontSize="14" size="30" cx={ cx(css.caption, this.getTextStyle(), uuiDataTableHeaderCell.uuiTableHeaderCaption) }>
-                        {this.props.column.caption}
+                    <Text
+                        key="text"
+                        lineHeight={ settings.sizes.dataTable.header.row.cell.columnCaption.lineHeight as TextProps['lineHeight'] }
```

Comment:
Probably we can move `settings.sizes.dataTable.header.row.cell.columnCaption.lineHeight` from settings to css variable?